server/pipeline/stepbuilder/stepBuilder.go

```diff
@@ -194,6 +195,20 @@ func (b *StepBuilder) genItemForWorkflow(workflow *model.Workflow, axis matrix.A
 		maps.Copy(item.Labels, b.DefaultLabels)
 	}
 
+	// "woodpecker-ci.org" namespace is reserved for internal use
```

Comment:
Does it need to have the `.org` suffix? I'd rather not have a valid URL in there but only the app name.

Reply:
Why do you think so? Its pretty common (e.g. `app.kubernetes.io` labels) and I would prefer it this way.

Reply:
app.kubernetes.io doesn't resolve to anything. Anyhow, no strong opinion.

The other comment is more important, though.